res/controllers/Roland_DJ-505-scripts.js

```diff
@@ -275,10 +273,11 @@ DJ505.browseEncoder = new components.Encoder({
     },
     input: function(channel, control, value, status, _group) {
         switch (status) {
-        case 0xBF: // Rotate.
-            var rotateValue = (value === 127) ? -1 : ((value === 1) ? 1 : 0);
+        case 0xBF: { // Rotate.
+            const rotateValue = (value === 127) ? -1 : ((value === 1) ? 1 : 0);
             this.onKnobEvent(rotateValue);
             break;
+        }
```

Comment:
the extra braces are weird, I assume this is from eslint?

Reply:
Without the extra braces, it causes an eslint error. Similar to c++, where declaring a variable inside a switch case without braces is also not allowed.

Reply:
right, makes sense. I just didn't know the extra braces were valid syntax.